app/views/overrides/activation_keys/_host_environment_select.html.erb

```diff
@@ -8,7 +8,7 @@
 
 <% env_select_id = using_hostgroups_page? ? :hostgroup_lifecycle_environment_id : :host_lifecycle_environment_id %>
 <% env_select_name =  using_hostgroups_page? ? 'hostgroup[lifecycle_environment_id]' : 'host[content_facet_attributes][lifecycle_environment_id]' %>
-<% env_select_attr = using_hostgroups_page? ? 'lifecycle_environment' : 'content_facet.lifecycle_environment' %>
+<% env_select_attr = using_hostgroups_page? ? 'single_lifecycle_environment' : 'content_facet.single_lifecycle_environment' %>
```

Comment:
The reference of `content_view` on the same page should probably be `single_content_view`.